src/assets/wise5/components/aiChat/aiChatService.ts

```diff
@@ -0,0 +1,29 @@
+import { Injectable } from '@angular/core';
+import { ComponentService } from '../componentService';
+import { AiChatMessage } from './aiChatMessage';
+
+@Injectable()
+export class AiChatService extends ComponentService {
+  getComponentTypeLabel() {
+    return $localize`AI Chat`;
+  }
+
+  createComponent(): any {
+    const component: any = super.createComponent();
+    component.type = 'AiChat';
+    component.model = 'gpt-4';
+    component.systemPrompt = '';
+    return component;
+  }
+
+  async sendChatMessage(messages: AiChatMessage[], model: string): Promise<any> {
```

Comment:
Define new class ```ChatMessageResponse``` to represent the response from the server? This will help make the code more clear.
```suggestion
  async sendChatMessage(messages: AiChatMessage[], model: string): Promise<ChatMessageResponse> {
```